sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/ExpressionLanguageInjection.kt

```diff
@@ -31,7 +31,7 @@ object ExpressionLanguageInjection {
      * Try to call the default constructor of the honeypot class.
      */
     private const val EXPRESSION_LANGUAGE_ATTACK =
-        "\${\"\".getClass().forName(\"$HONEYPOT_CLASS_NAME\").newInstance()}"
+        "\${\"\".getClass().forName(\"$HONEYPOT_CLASS_NAME\").getMethod(\"el\").invoke(null)}"
```

Comment:
Why can't we keep `newInstance` here? It is shorter and should also exercise non-trivial functionality (constructor call).

Reply:
This was my first attempt, but I couldn't get the string size to be less than 64 bytes while using the constructor that sets the sanitizer ID. The default constructor sets the sanitizer ID to the RCE sanitizer.

Do you think we can reduce the attack string size to less than 64 bytes? `"\${\"\".getClass().forName(\"$HONEYPOT_CLASS_NAME\").getDeclaredConstructor(byte.class).newInstance((byte)2)}"`

Reply:
I tried but didn't find a way. You can get rid of the `(byte)` by changing the constructor parameter type to `int` and also use `getConstructors`, but that's still off by a large margin.

I only found this improvement over the current approach:
```suggestion
        "\${Byte.class.forName(\"$HONEYPOT_CLASS_NAME\").getMethod(\"el\").invoke(null)}"
```

Reply:
Good idea. Also added a check for the size of the payloads we use in the deserialization sanitizer